pages/common/git-bisect.md

```diff
@@ -0,0 +1,20 @@
+# git bisect
+
+> Use binary search to find the commit that introduced a bug.
+> Git automatically jumps back and forth in the commit graph to progressively narrow down the faulty commit.
+
+- Start a bisect section by passing a known good commit and a known bad commit:
+
+`git bisect start {{bad_commit}} {{good_commit}}`
+
+- For each commit that `git bisect` selects, mark it as "bad" or "good":
+
+`git bisect {{good|bad}}`
+
+- After `git bisect` pinpoints the faulty commit, end the bisect session and return to the previous branch:
+
+`git bisect reset`
+
+- Skip a commit during a bisect (e.g. one that causes a broken build due to another issue):
```

Comment:
If a commit is broken for a different reason, you may not be able to tell whether the issue you're looking for is present in it or not -- the idea here is to tell git bisect to ignore this commit rather than pass it as a data point, as that could be a false positive or a false negative, both of which would pollute the binary search process.